modules/hitl2/src/views/full/AgentStatus.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext, useEffect } from 'react'
+
+import { SocketMessageType } from './../../types'
+import { Context, Store } from './agentStatus/Store'
+import { Api } from './Api'
+import AgentIcon from './Components/AgentIcon'
```

Comment:
@MaxCloutier There is already an `Icons` directory. Feel free to migrate icons into the main `Components` directory or to leave them in the `Icons`, but could you please choose a single option? Thanks!

Reply:
that's not the icon, but the component displaying the icon and the online dot 
![Screen Shot 2020-10-23 at 7 26 17 AM](https://user-images.githubusercontent.com/3555826/97016014-149e4780-1501-11eb-90cc-e5702f7567ed.png)